### DISCUSSION TITLE
Rider shows different coverage than coverlet

### DISCUSSION THREAD

**Discussion by DefinitelyADev** [coverlet-coverage/coverlet#1615] [Q&A]
I've been trying to find the reason to this for hours now and I'm running out of options...
Rider reports 79% coverage and coverlet reports 38%. This is a substantial difference. Has anyone faced a similar issue before?
I understand they might collect coverage differently but the numbers are too different. Is there a way to make them consistent at least?

**Comment by Bertk** [comment]
We do not use Rider and have no experience with this tool. Maybe you need to use additional filters to exclude

- generated code
- test code
- code from other projects/packages

Did you compare code coverage reports created with [ReportGenerator](https://github.com/danielpalme/ReportGenerator) for Rider and coverlet results.
